grammar/cypher.xml

```diff
@@ -89,6 +84,51 @@
     </alt>
   </production>
 
+  <production name="SingleQuery" rr:inline="true">
+    <alt>
+      <seq>
+        <non-terminal ref="StartingClause"/> &WS;
+        <repeat><non-terminal ref="Clause"/> &WS;</repeat>
```

Comment:
I don't think we should allow _any_ clause in the middle of this sequence. For example once you hit `Return` it is over.